sunpy/visualization/mapsequenceanimator.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
```

Comment:
this file has moved in master? Also shouldn't we be deprecating the old one?

Reply:
Actually `MapCube.peek` can just return an instance of `MapSequenceAnimator` we don't need to keep both implementations around. just do a `MapCubeAnimator  = MapSequenceAnimator` for namespace compatibility.